tests/dummy/app/components/esri-map.js

```diff
@@ -1,6 +1,6 @@
 import Component from '@ember/component';
 
-import Map from 'esri/Map';
+import Map from 'esri/map';
```

Comment:
I found the dummy to be broken when I first tested my changes - the changes I made here were the minimal to get it to build/load.